docs/commands/move.md

```diff
@@ -0,0 +1,33 @@
+# `move` command
+
+Note: The implementation for `copy` and `move` are exactly the same. The only different is that move will remove the source after a successful copy.
+
+The `move` commands works like the `mv` or `rsync` binaries. It allows moving either single entries or whole folders around. Moving across mounts is supported.
```

Comment:
"command", singular.
proposition:  works like the Unix `mv` or `rsync` binaries.